apollo-federation/src/merge.rs

```diff
@@ -510,6 +515,17 @@ impl Merger {
                     },
                 );
 
+                let overrides_directive_option = Option::and_then(
+                    field.directives.get_all(&override_directive_name).next(),
```

Comment:
Curious why this vs

`field.directives.get_all(&override_directive_name).next().and_then(...)1

Reply:
just how it was already written! i can definitely change it

Reply:
lol Option::and_then avoided a clippy warning